packages/nois-protocol/src/ibc_msg.rs

```diff
@@ -10,15 +10,34 @@ pub enum InPacket {
     RequestBeacon {
         /// Beacon publish time must be > `after`
         after: Timestamp,
+        /// The callback data set by the proxy in a proxy specific format.
+        callback: Binary,
+    },
+    RequestScheduleJob {
```

Comment:
I think we should have a corresponding `OutPacket::DeliverScheduleJob` case which does not contain the randomness field